sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -172,6 +191,39 @@ export interface MetricNamespace {
     type?: string;
 }
 
+// @public (undocumented)
+export interface MetricResultsResponseValuesItem {
+    endtime: string;
+    interval?: string;
+    namespace?: string;
+    resourceid?: string;
+    resourceregion?: string;
+    starttime: string;
```

Comment:
Do we not want the starttime and endtime to be an actual timestamp - DateTime ?

Reply:
Also should we rename :
starttime => startTime
endtime => endTime
resourceregion => resourceRegion

Reply:
They are string based on spec